tensorboard/program.py

```diff
@@ -727,17 +728,27 @@ def handle_error(self, request, client_address):
             logger.error("HTTP serving error", exc_info=exc_info)
 
     def get_url(self):
-        if self._auto_wildcard:
-            display_host = socket.gethostname()
-        else:
-            host = self._host
-            display_host = (
-                "[%s]" % host
-                if ":" in host and not host.startswith("[")
-                else host
-            )
+        if not self.display_host:
+            if self._auto_wildcard:
+                self.display_host = socket.getfqdn()
+
```

Comment:
nit: can we leave out the blank lines inside the function, since it's not that long?

If we want separation for clarity I'd recommend just factoring out a private helper function as described in the previous comment.

Reply:
Do you mean just removing line 734, which I did in f3bbb87? If not, I don't see you mean by "as described in the previous comment".